components/src/lib/modal/Modal.stories.tsx

```diff
@@ -0,0 +1,159 @@
+import Modal from "./Modal";
+
+import { StoryObj, Meta } from "@storybook/react";
+import { fn } from "@storybook/test";
+
+const meta = {
+  title: "Components/Modal",
+  component: Modal,
+  parameters: {
+    layout: "centered",
+    docs: {
+      description: {
+        component: "모달 컴포넌트",
+      },
+    },
+  },
+  argTypes: {
+    isOpen: {
+      control: "boolean",
+      default: true,
+      description: "모달의 상태",
+      table: {
+        type: {
+          summary: "boolean",
+        },
+      },
+    },
+    onClose: {
+      description: "modal을 열고 닫기 위한 핸들러 함수",
+    },
+    style: {
+      control: "object",
+      table: {
+        type: {
+          summary: "object",
+        },
+      },
+      description: "모달 스타일을 자유롭게 정의하는 속성",
+    },
+    position: {
+      control: { type: "radio" },
+      options: ["top", "bottom", "center"],
+      description: "모달 위치를 페이지 상단, 중앙, 하단으로 선택하는 속성",
+    },
+  },
+  args: {
+    onClose: fn(),
+  },
+  render: ({ style, ...args }) => {
+    return (
+      <Modal style={style} {...args}>
+        <Modal.ModalContent style={style}>
+          <span>올리와 썬데이의 기본 모달</span>
+        </Modal.ModalContent>
+      </Modal>
+    );
+  },
+  tags: ["autodocs"],
+} satisfies Meta<typeof Modal>;
+
+export default meta;
+
+type Story = StoryObj<typeof Modal>;
+
+export const 기본: Story = {
+  parameters: {
+    docs: {
+      description: {
+        story: "내용만 있는 기본 모달",
+      },
+    },
+  },
+  args: {
+    isOpen: true,
+    position: "center",
+  },
+};
+
+export const 제목이있는모달: Story = {
```

Comment:
띄어쓰기를 하고 싶은 곳에 언더바를 넣으면 스토리북에서는 띄어쓰기로 보인답니다!
```suggestion
export const 제목이_있는_모달: Story = {
```

Reply:
혹은 스토리에 속성으로 name으로 넣어줘도 되고요!
[이 문서](https://storybook.js.org/docs/api/csf#named-story-exports)를 참고해봐도 됩니다!